kuma/urls.py

```diff
@@ -89,6 +90,12 @@ def serve_from_media_root(request, path):
 urlpatterns += i18n_patterns(url(r'^docs.json$', document_as_json,
                                  name='wiki.json'))
 urlpatterns += i18n_patterns(url(r'^docs/', include(wiki_lang_urlpatterns)))
+
+# A special endpoint for our React spike, local and staging only
+if settings.DOMAIN != settings.PRODUCTION_DOMAIN:
```

Comment:
I double-checked, this will work correctly - it will show up in staging and development, but will be a 404 in production.